oap-server/server-library/library-client/src/main/java/org/apache/skywalking/oap/server/library/client/elasticsearch/ElasticSearchClient.java

```diff
@@ -292,7 +300,34 @@ public int delete(String indexName, String timeBucketColumnName, long endTimeBuc
 
     public String formatIndexName(String indexName) {
         if (StringUtils.isNotEmpty(namespace)) {
-            return namespace + "_" + indexName;
+            return namespacePrefix + indexName;
+        }
+        return indexName;
+    }
+
+    private JsonObject undoFormatIndexName(JsonObject index) {
```

Comment:
We don't need this method, this should move to test class only. Right?

Reply:
done